### ISSUE TITLE
Identity Manager cannot create identity using did from `DidDht`

### ISSUE THREAD

**Issue by xiaoyunyang** [TBD54566975/web5-js#440]
When I was implementing an identity manager the latest `IdentityAgent` from `@web5/identity-agent`, I ran into a problem trying to pass a `did` created using `DidDht` into `identityManager.create()` function. Specifically, the create function expects a different `PortableDid` type than the one that you can create using `DidDht`.

Here's the code I have

```ts
import { IdentityAgent } from "@web5/identity-agent";
import { DidDht } from "@web5/dids";

const didDht = await DidDht.create();
const portableDid = await didDht.export();

const agent = await IdentityAgent.create();

const identity = await agent.identityManager.create({
  name,
  kms: "local",
  did: portableDid, // <-- error occurs here
});
```

The error is

```
No overload matches this call.
  Overload 1 of 2, '(options: CreateWithDidMethod<keyof CreateDidMethodOptions>): Promise<ManagedIdentity>', gave the following error.
    Object literal may only specify known properties, and 'did' does not exist in type 'CreateWithDidMethod<keyof CreateDidMethodOptions>'.
  Overload 2 of 2, '(options: CreateWithDid): Promise<ManagedIdentity>', gave the following error.
    Type 'PortableDid' is missing the following properties from type 'PortableDid': did, keySet
```

I'm using these versions

```json
{
  "@web5/dids": "^0.4.1",
  "@web5/identity-agent": "^0.2.6",
}
```

**Comment by xiaoyunyang** [comment]
I think the problem is the agent package uses `@web5/dids` v0.2.4 which has a different `PortableDid` definition. 

https://github.com/TBD54566975/web5-js/blob/9c27ee47777b2622003e50c89f29dd17c1153229/packages/agent/package.json#L75

Can this be upgraded to the latest version?

**Comment by frankhinek** [comment]
@xiaoyunyang Thanks for opening the issue. This will be resolved in the `0.9` release of `@web5/api` and `0.3` of `@web5/identity-agent`.
